cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -178,6 +180,8 @@ message CassandraNode {
 
     optional int32 cassandraDaemonPid = 12;
 
+    optional string replacementForIp = 13;
```

Comment:
It's not tracked long-term. Once the node has been replaced, the field is cleared.